readme.md

```diff
@@ -9,26 +9,18 @@
 - Explain the benefits of jQuery
 - Explore jQuery methods for DOM manipulation and traversal
 
-## Intro (5 minutes)
+## Framing (5 minutes / 0:05)
 
-We learned some things about JS objects. We learned that we can give objects key value pairs to program logic from real life objects. Turns out, we can actually treat elements in our HTML as objects in our JS programs.
+In the Objects & Functions lesson you learned about objects as data structures and how we can use them to store data and methods. Today we will learn about how Javascript uses objects to represent what you see in the browser.
 
```

Comment:
- comma after "lesson" in the first sentence
- comma after "Today" in the second